Source/ThirdParty/ANGLE/extensions/ANGLE_variable_rasterization_rate_metal.txt

```diff
@@ -0,0 +1,62 @@
+Name
+
+    ANGLE_variable_rasterization_rate_metal
+
+Name Strings
+
+    GL_ANGLE_variable_rasterization_rate_metal
+
+Contributors
+
+    Dan Glastonbury, Apple Inc.
+
+Contact
+
+    Dan Glastonbury, Apple Inc. (djg 'at' apple 'dot' com)
+
+Status
+
+    Incomplete
+
+Version
+
+    Last Modified Date: March 30, 2023
+    Author Revision: 1
+
+Number
+
+    OpenGL ES Extension XX
+
+Dependencies
+
+Overview
+
+IP Status
+
+    No known IP claims.
+
+New Types
+
+    /*
+     * GLMTLRasterizationRateMapANGLE is an opaque handle to an id<MTLRasterizationRateMap>
+     */
+    typedef void* GLMTLRasterizationRateMapANGLE;
+
+New Procedures and Functions
+
+    void BindMetalRasterizationRateMapANGLE(GLMTLRasterizationRateMapANGLE map);
```

Comment:
Though this extension's incomplete and being prototyped, would you like to mention anything about the semantics of this new binding point? Should this define ownership of the MTLRasterizationRateMap? MTLRasterizationRateMap needs to define multiple layer maps if performing layered rendering - which I assume would mean using multiple draw buffers in OpenGL ES semantics. Should this mention errors, for example that subsequent draw calls will fail with INVALID_OPERATION if the number of color attachments on the bound framebuffer doesn't match the number of layer maps in the rasterization rate map? Does passing nullptr restore the GL state to its original value?